test/chrono-test.cc

```diff
@@ -49,7 +49,14 @@ std::string system_strftime(const std::string& format, const std::tm* timeptr,
   os.imbue(loc);
   facet.put(os, os, ' ', timeptr, format.c_str(),
             format.c_str() + format.size());
+#ifdef _WIN32
+  // Workaround to early ucrt bug
```

Comment:
early?

Reply:
Early version.

I have not tested all versions. But in the latest version of the Windows SDK I have installed with MSVS, '%z' returns '+0000'. In the old version: '-0000'.

If remove platform-dependent code, this change will not be necessary.

Reply:
Since we are keeping this, let's tweak it a bit for clarity:

```c++
// Workaround a bug in older versions of ucrt.
```

BTW is "ucrt" the correct term here? Is it Microsoft C runtime library? Maybe calling it Microsoft CRT here would be clearer?